pkg/version/version.go

```diff
@@ -33,6 +35,24 @@ var (
 	ReleaseStatus = "unreleased"
 )
 
+func GetMajorVersion() string {
+	v, err := semver.Parse(Version[1:])
+	if err != nil {
+		println(err)
```

Comment:
why not using log but output directly?

Reply:
Since the `Version` is basically a constant, I think we could just ignore the error. Let me remove this error handling.

Reply:
I removed `GetMajorVersion` and `GetMinorVersion` and make the `GetVersion` to return a semantic version object.